v19.2/performance-benchmarking-with-tpc-c-local.md

```diff
@@ -0,0 +1,171 @@
+1. Use the [`cockroach start`](start-a-node.html) command to start 3 nodes:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cockroach start \
+    --insecure \
+    --store=tpcc-local1 \
+    --listen-addr=localhost:26257 \
+    --http-addr=localhost:8080 \
+    --join=localhost:26257,localhost:26258,localhost:26259 \
+    --background
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cockroach start \
+    --insecure \
+    --store=tpcc-local2 \
+    --listen-addr=localhost:26258 \
+    --http-addr=localhost:8081 \
+    --join=localhost:26257,localhost:26258,localhost:26259 \
+    --background
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cockroach start \
+    --insecure \
+    --store=tpcc-local3 \
+    --listen-addr=localhost:26259 \
+    --http-addr=localhost:8082 \
+    --join=localhost:26257,localhost:26258,localhost:26259 \
+    --background
+    ~~~
+
+2. Use the [`cockroach init`](initialize-a-cluster.html) command to perform a one-time initialization of the cluster:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cockroach init \
+    --insecure \
+    --host=localhost:26257
+    ~~~
+
+## Step 2. Load data for the benchmark
+
+CockroachDB comes with built-in load generators for simulating different types of client workloads, printing out per-operation statistics every second and totals after a specific duration or max number of operations. This step features CockroachDB's version of the TPC-C workload.
+
+Use [`cockroach workload`](cockroach-workload.html) to load the initial schema and data:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach workload init tpcc \
+--warehouses=10 \
+'postgresql://root@localhost:26257?sslmode=disable'
+~~~
+
+This will take about ten minutes to load, after which you'll see the following output:
+
+~~~
+I191024 03:41:34.308865 1 workload/workloadsql/dataload.go:135  imported warehouse (0s, 10 rows)
+I191024 03:41:34.353839 1 workload/workloadsql/dataload.go:135  imported district (0s, 100 rows)
+I191024 03:42:00.865733 1 workload/workloadsql/dataload.go:135  imported customer (27s, 300000 rows)
+I191024 03:42:13.233536 1 workload/workloadsql/dataload.go:135  imported history (12s, 300000 rows)
+I191024 03:42:20.893806 1 workload/workloadsql/dataload.go:135  imported order (8s, 300000 rows)
+I191024 03:42:21.716409 1 workload/workloadsql/dataload.go:135  imported new_order (1s, 90000 rows)
+I191024 03:42:23.483713 1 workload/workloadsql/dataload.go:135  imported item (2s, 100000 rows)
+I191024 03:43:37.660918 1 workload/workloadsql/dataload.go:135  imported stock (1m14s, 1000000 rows)
+I191024 03:46:51.682670 1 workload/workloadsql/dataload.go:135  imported order_line (3m14s, 3001222 rows)
+~~~
+
+## Step 3. Run the benchmark
+
+Run the workload for ten "warehouses" of data for five minutes (300 seconds):
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach workload run tpcc \
+--warehouses=10 \
+--ramp=30s \
```

Comment:
We should probably run this longer. How about a `3m` ramp and a `10m` run?

Reply:
ack